### ISSUE TITLE
[Security] De-scope Vision Zero CI access permissions

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#11151]
1. Identify the minimum AWS permissions needed for the Vision Zero CI
2. Adjust the AWS user permissions for the VZ CI users to match ☝️

**Comment by johnclary** [comment]
@mddilley @patrickm02L FYI

**Comment by frankhereford** [comment]
@frankhreford reminder for myself to find the sibling issue to this for Moped that John took on for inspiration

**Comment by mddilley** [comment]
Starting working on this today while rotating some secrets, and I created a IAM policy named `atd-zappa` which has the necessary permissions for Zappa deploy and update. The IAM user also needs permissions for the SQS part of the flow so I've added full permissions for SQS for now.

Permissions outside of that needed by the flow were:
- `lambda:TagResource`
- `lambda:ListEventSourceMappings`

I added these to the Zappa policy for now, but I could probably come up with a better way to organize these. I'll keep working on this.

**Comment by mddilley** [comment]
Tested the CI in staging (CircleCI and Github Action), and they both worked. Sticking with a Zappa IAM policy that can be used for both Moped and VZ. I ran into one security advisory from AWS in that policy, and I've made an issue to address that in the future. 
https://github.com/cityofaustin/atd-data-tech/issues/11934

The permissions for this IAM user now only include:
- everything covered in the Zappa policy
- SQS

This is a big step forward, but I created https://github.com/cityofaustin/atd-data-tech/issues/11943 to further refine these permissions if we agree to do that.
